### ISSUE TITLE
Hello, could I ask why I got totally different output when I run predict_mlsp_wide.ipynb?

### ISSUE THREAD

**Issue by tsz12** [subpic/ava-mlsp#11]
<img width="372" alt="4f267ab212970a42da382282083a73b" src="https://user-images.githubusercontent.com/65204135/200104267-c75b701d-180d-41b8-968d-52171925f589.png">

**Comment by subpic** [comment]
Hi,

I can't really say what's going on. Maybe you can confirm the following:
- Did you run the code on colab? 
- Which image did you feed the model, was it the same as in the demo code?
- Was its normalized the same as in the demo?
- Did you use the an appropriate version of TF in case something changed since the demo was released?

Best,
Vlad

**Comment by Wunaiq** [comment]
> 

I met the same situation.
Have you found out the reason, please?

**Comment by Wunaiq** [comment]
I got such an hint during the script is running:

```
2023-02-01 12:00:27.924039: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 523 MB memory) -> phy
sical GPU (device: 1, name: GeForce GTX 1080 Ti, pci bus id: 0000:03:00.0, compute capability: 6.1)
2023-02-01 12:00:27.924357: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:1005] successful NUMA node read from SysFS had negative value (-1), but there must be at least one N
UMA node, so returning NUMA node zero
2023-02-01 12:00:27.924775: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:2 with 10419 MB memory) -> p
hysical GPU (device: 2, name: GeForce GTX 1080 Ti, pci bus id: 0000:05:00.0, compute capability: 6.1)
Creating multi-pooled model

Model NOT loaded: models/irnv2_mlsp_wide_orig/model_best_weights.h5 does not exist
```

I guess that the model outputs wrong results because it did not load the pretrained weights correctly.
But I just don't know why the pretrained weights cannot be loaded and I am sure that I have downloaded the weight and put it into the correct folder.

**Comment by WeitaoVan** [comment]
@Wunaiq hi, did you solve the problem？ I met the same problem. But the log shows it successfully loaded the model weights.
It's strange that the output score is a negative value.
![log screenshot](https://user-images.githubusercontent.com/16603489/230842782-55fdfb67-fc47-48ef-b1ab-eaaaf43a46ca.png)

**Comment by WeitaoVan** [comment]
> 
The input image is the one in the repo's `images/` folder

**Comment by yushilinnn** [comment]
@WeitaoVan @tsz12 @Wunaiq 
Has anyone solved the problem of negative value? I'm pretty sure I used the environment provided by the author (tensorflow-gpu 1.14.0, keras-gpu 2.2.4) and successfully loaded the model.
When I use it on one computer, it is normal. However, when I use another computer, I get a negative value. The features extracted by the two computers are also different.
If anyone can provide a solution that would be greatly appreciated.
